src/main/java/frc/robot/commands/AutomaticShooter.java

```diff
@@ -243,14 +259,14 @@ public void getDistance()
     {
       //todo: Use vision and network tables to get targetDistance
       targetDistance = visionDistance.getDouble(-1.0);
-      System.out.println("targetDistance from vision: "+targetDistance);
+      //System.out.println("targetDistance from vision: "+targetDistance);
       //@todo: Double check if the target distance is valid or not.
       //Note: Unit is feet, we have to convert to meter
 
     }
 
     //@todo: validate the targetDistance
-    if(targetDistance > 5 || targetDistance < 0.5)
+    if(targetDistance > 20 || targetDistance < 0.5)
```

Comment:
I don't think vision is able to see the target closer than 6 feet (would have to quickly test on the robot to see its range to get exact number).   Distance greater than 20 is no able to be seen, actually, it might even be 18.   The angle of the camera is quite severe so we lose some of the further distance due to the cameras field of view.